apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx

```diff
@@ -47,15 +59,18 @@ const UniqueDefenders: React.FC<
       </Text>
       {uniqueDefenders.map((defender, index) =>
         defender?.name ? (
-          <Box display="flex" key={defender.name}>
-            <Text>
-              {`${defender.name}${defender.email ? `, ${defender.email}` : ''}${
-                defender.phoneNumber ? `, s. ${defender.phoneNumber}` : ''
-              }`}
+          <Box display="flex" key={defender.name} role="paragraph">
```

Comment:
Should we hide this if the user is a defender so we're not displaying his own email to him for no reason?

Reply:
I think we should leave it as is. You could make the case that it's good for defenders to see that the correct email and phone number is set in RVG..